static/templates/pm_list_item.hbs

```diff
@@ -3,13 +3,9 @@
 
         <div class="pm_left_col">
             {{#if is_group}}
-                {{#if fraction_present}}
-                <span class="{{user_circle_class}} user_circle" style="background:hsla(106, 74%, 44%, {{fraction_present}});"></span>
-                {{else}}
-                <span class="{{user_circle_class}} user_circle" style="background:none; border-color:hsl(0, 0%, 50%);"></span>
-                {{/if}}
+            <span class="show_user_count">{{show_user_count}}</span>
```

Comment:
Let's call the class and value `group_user_count` or something like that -- the `show_` in the name is kinda confusing.